src/actions/Action.ts

```diff
@@ -32,8 +34,9 @@ class Action implements IAction {
   addFlag(flag: Flag): this {
     const existingFlag = this.params.get('fl_');
     if (existingFlag){
-      existingFlag.addValue(flag.paramValue);
+      existingFlag.addValue(flag.paramValue.setDelimiter('.'));
```

Comment:
Seems like we do this regardless of the if branches, can this be moved to the top?

`flag.paramValue.setDelimiter('.')`